<ui version="4.0" >
 <class>FormatSettingsDialog</class>
 <widget class="QDialog" name="FormatSettingsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>51</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Custom settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QCheckBox" name="saveCB" >
       <property name="text" >
        <string>Save as default settings</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButotn" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>FormatSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>400</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel" >
     <x>224</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButotn</sender>
   <signal>clicked()</signal>
   <receiver>FormatSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>317</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel" >
     <x>224</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
